sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlockBlobClient.java

```diff
@@ -135,7 +135,6 @@ public BlockBlobItem upload(InputStream data, long length) throws IOException {
      * @param context Additional context that is passed through the Http pipeline during the service call.
      *
      * @return The information of the uploaded block blob.
-     * @throws UnexpectedLengthException when the length of data does not match the input {@code length}.
```

Comment:
We should suppress the Javadoc issue, this will through this type of exception and we shouldn't hide that if the linting tools can't figure that out.

Reply:
Fixed in this PR: https://github.com/Azure/azure-sdk-for-java/pull/5245/files

Reply:
https://stackoverflow.com/questions/27938039/unable-to-get-class-information-for-checkstyle